.codeclimate.yml

```diff
@@ -25,7 +25,7 @@ checks:
     enabled: true
 plugins:
   rubocop:
-    enabled: true
-    channel: rubocop-0-80
+    enabled: false
+    channel: rubocop-0-67
     config:
-      file: .rubocop.yml
```

Comment:
This doesn't seem right. `rubocop.yml` includes `rubocop_todo.yml`.

Reply:
Woops, forgot to undo that change